### DISCUSSION TITLE
Instalar MAP-OS usando IIS

### DISCUSSION THREAD

**Discussion by labinfatec** [RamonSilva20/mapos#1757] [General]
Gostaria de saber se é possível instalar o MAP-OS usando IIS, se teria algum tutorial de como proceder, ao invés de usar o XAMPP.

**Comment by Pr3d4dor** [comment]
@labinfatec Nunca instalei o mapos usando o IIS, mas creio que deva ser o mesmo processo de subir qualquer aplicação de PHP usando IIS

**Comment by ClaudineySR** [comment]
Faça a instação localmente com o XAMPP depois copie a pasta do MAP-OS  para a pasta do site no iis, configure os arquivos  config.php e  database.php na pasta \application\config
